transaction/core/src/constants.rs

```diff
@@ -22,8 +22,11 @@ pub const MAX_TOMBSTONE_BLOCKS: u64 = 100;
 /// The MobileCoin network will contain a fixed supply of 250 million mobilecoins (MOB).
 pub const TOTAL_MOB: u64 = 250_000_000;
 
+/// one milliMOB = 1e9 picoMOB
+pub const MILLIMOB: u64 = 1_000_000_000;
```

Comment:
This should probably be named something more like `MILLIMOB_FROM_PICOMOB` (or `PICOMOB_TO_MILLIMOB`).